src/main.cpp

```diff
@@ -337,6 +338,17 @@ static void parse_env(BPFtrace& bpftrace)
                     "BPFTRACE_MAX_MAP_KEYS instead.";
     config_setter.set(ConfigKeyInt::max_map_keys, x);
   });
+
+  get_bool_env_var("BPFTRACE_USE_BLAZESYM", [&](bool x) {
```

Comment:
We don't need t make this an env var too. Let's just leave it as a config.

Reply:
See my comment below, I don't plan on us going back and forth on symbolication strategy. I seriously doubt that blazesym won't be significantly better than what we have now.

Reply:
Okay, but somebody may do; at the very least I know I do. Given that we will have both in place for a while, I very much believe that it makes sense to make it easy to switch to compare, from an engineering perspective. Put it differently: what is the problem with these five odd lines of code that makes them so troublesome to include?

Reply:
Ok, if it's helping with development (and not surfaced to users) then we can leave it. Let's just make a note to yank it out when blazesym is fully integrated and the default.